ui/imports/shared/controls/qmldir

```diff
@@ -18,6 +18,7 @@ InformationTag 1.0 InformationTag.qml
 InformationTile 1.0 InformationTile.qml
 Input 1.0 Input.qml
 LoadingTokenDelegate 1.0 LoadingTokenDelegate.qml
+LinkPreviewDebugView 1.0 LinkPreviewDebugView.qml
```

Comment:
Just wondering how is this view trigerred? :)

Reply:
I just used it a lot during debugging. And I think that we will keep using it for some time, until preview cards are stable enough. Because it's hard to distinguish if, for example, a `DisplayName` is missing because of a bug in backend or UI 😄 

Right now it's not used anywhere.